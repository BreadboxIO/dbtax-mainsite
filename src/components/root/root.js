import { HashRouter } from 'react-router-dom';

import React from 'react';
import { ROUTES_CONFIG } from 'constants/routes';
import { Provider } from 'react-redux';
import { App } from 'components/app/app';
import PropTypes from 'prop-types';
import { RouteWithSubRoutes } from './route-with-sub-routes/route-with-sub-routes';

export const Root = props => {
  const { store } = props;

  return (
    <Provider store={store}>
      <HashRouter hashType="noslash">
        <App>
          {Object.keys(ROUTES_CONFIG).map(key => {
            const route = ROUTES_CONFIG[key];
            const { component, exact, path, routes } = route;

            return (
              <RouteWithSubRoutes
                component={component}
                exact={exact}
                key={key}
                path={path}
                routeKey={key}
                routes={routes}
              />
            );
          })}
        </App>
      </HashRouter>
    </Provider>
  );
};

Root.propTypes = {
  store: PropTypes.object
};

Root.defaultProps = {
  store: null
};
