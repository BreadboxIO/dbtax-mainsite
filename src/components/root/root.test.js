import { Root } from './root';
/* global React, create */

describe('<Root />', () => {
  it('should render correctly', () => {
    const component = create(<Root />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
