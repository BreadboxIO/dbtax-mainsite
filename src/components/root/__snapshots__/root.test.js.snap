// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Root /> should render correctly 1`] = `
<div
  className="root"
>
  <div
    className="root"
  >
    <div
      className="max-width--xl margin--horizontal--auto"
    >
      <a
        className="animated fadeIn"
        href="tel:+14147647730"
        rel="noopener noreferrer"
        target="_blank"
      >
        <i
          className="fas fa-phone-alt"
        />
         
        414-764-7330
      </a>
      <a
        className="animated fadeIn margin--left--half"
        href="https://www.google.com/maps/place/1125+Milwaukee+Ave,+South+Milwaukee,+WI+53172"
        rel="noopener noreferrer"
        target="_blank"
      >
        <i
          className="fas fa-home"
        />
         
        1125 Milwaukee Ave.
        <span
          className="display--xs--none display--md--inline-block"
        >
           
          · 
          South Milwaukee
          , 
          WI
        </span>
         
        · 
        53172
      </a>
    </div>
  </div>
  <div
    className="root"
  >
    <div
      className="content"
    >
      <div
        className="logoContainer"
      >
        <a
          className="display--block"
          href="#"
          onClick={[Function]}
        >
          <img
            alt="DB Tax & Financial Services"
            className="logo"
            src="db-tax-logo.png"
          />
        </a>
      </div>
      <nav
        className="links"
      >
        <a
          aria-current="page"
          className="active"
          href="#"
          onClick={[Function]}
          style={Object {}}
        >
          About
        </a>
        <a
          aria-current="page"
          className="active"
          href="#"
          onClick={[Function]}
          style={Object {}}
        >
          Services
        </a>
        <a
          aria-current="page"
          className="active"
          href="#"
          onClick={[Function]}
          style={Object {}}
        >
          Client Resources
        </a>
        <a
          aria-current="page"
          className="active"
          href="#"
          onClick={[Function]}
          style={Object {}}
        >
          Legal
        </a>
      </nav>
      <div
        className="accountAccess"
      >
        <button
          className="root button"
          onClick={null}
          type="button"
        >
          Account Access
        </button>
      </div>
      <div
        className="mobileToggle"
      >
        <button
          className="root root"
          onClick={[Function]}
          type="button"
        >
          <i
            className="fas fa-bars"
          />
        </button>
      </div>
    </div>
  </div>
  <div
    className="animated fadeIn root"
  >
    <div
      className="root"
    >
      <div
        className="image"
        style={
          Object {
            "backgroundImage": "url(woman-using-financial-app.jpg)",
          }
        }
      />
      <div
        className="spacer"
      />
      <div
        className="content"
      >
        <div
          className="max-width--sm"
        >
          <div
            className="root"
          >
            <div
              className="subtitle color--primary"
            >
              Client Resources
            </div>
            <h1>
              Don't buy the hype.
            </h1>
          </div>
          <p>
            There are a lot of misconceptions about Social Security. Here's the truth about three of them.
          </p>
          <button
            className="root display--inline-block"
            onClick={null}
            type="button"
          >
            Learn More
          </button>
        </div>
      </div>
    </div>
    <div
      className="root primary-light"
    >
      <div
        className="content"
      >
        <div
          className="max-width--sm margin--horizontal--auto text-align--center"
        >
          <div
            className="root"
          >
            <div
              className="subtitle color--secondary"
            >
              Services
            </div>
            <h1>
              We're here to help you reach your financial success
            </h1>
          </div>
          <p
            className="color--quaternary-light margin--bottom"
          >
            We want to learn more about your personal situation, identify your dreams and goals, and understand your tolerance for risk.
          </p>
        </div>
        <div
          className="root"
        >
          <a
            className="root"
            href="#"
            onClick={[Function]}
          >
            <div
              className="iconContainer"
            >
              <i
                className="icon fas fa-coins"
              />
            </div>
            <h4
              className="color--type"
            >
              Taxes
            </h4>
            <p>
              We provide both businesses or persons with the professional taxation expertise and know how that they deserve year round.
            </p>
          </a>
          <a
            className="root"
            href="#"
            onClick={[Function]}
          >
            <div
              className="iconContainer"
            >
              <i
                className="icon fas fa-money-check-alt"
              />
            </div>
            <h4
              className="color--type"
            >
              Accounting
            </h4>
            <p>
              We provide our clients with value added accounting and outsourcing services for their businesses.
            </p>
          </a>
          <a
            className="root"
            href="#"
            onClick={[Function]}
          >
            <div
              className="iconContainer"
            >
              <i
                className="icon fas fa-search-dollar"
              />
            </div>
            <h4
              className="color--type"
            >
              Financial Services
            </h4>
            <p>
              We will ensure that your financial records are maintained correctly and accurately.
            </p>
          </a>
        </div>
      </div>
    </div>
    <div
      className="root root primary"
    >
      <div
        className="content"
      >
        <div
          className="content"
        >
          <div
            className="primary"
          >
            <div
              className="root"
            >
              <div
                className="subtitle color--secondary"
              >
                About
              </div>
              <h1>
                Mission Statement
              </h1>
            </div>
            <p>
              Our mission is to get to know and understand your needs, wants, and long-term goals. We want to help you develop, implement, and monitor a strategy that’s designed to address your individual situation.
            </p>
            <p>
              We understand the challenges families face today.
            </p>
            <p>
              From managing debt to saving for college to retirement, these personal finance challenges can be overwhelming. Our commitment is to utilize all of our resources to help you pursue your goals.
            </p>
            <p>
              We believe in thinking “out of the box” and we are not afraid to challenge conventional wisdom in our approach to investing and preserving wealth. All of our energy, commitment, and efforts are focused on you, the client, and your satisfaction.
            </p>
          </div>
          <div
            className="imageContainer"
          >
            <div
              className="image"
              style={
                Object {
                  "backgroundImage": "url(woman-man-financial-consulting.jpg)",
                }
              }
            />
          </div>
        </div>
      </div>
    </div>
    <div
      className="root"
    >
      <div
        className="map"
      >
        <div
          onClick={[Function]}
          onMouseDownCapture={[Function]}
          onMouseMove={[Function]}
          style={
            Object {
              "height": "100%",
              "margin": 0,
              "padding": 0,
              "position": "relative",
              "width": "100%",
            }
          }
        >
          <div
            style={
              Object {
                "height": "100%",
                "left": 0,
                "margin": 0,
                "padding": 0,
                "position": "absolute",
                "top": 0,
                "width": "100%",
              }
            }
          />
          <div
            style={
              Object {
                "height": "50%",
                "left": "50%",
                "margin": 0,
                "padding": 0,
                "position": "absolute",
                "top": "50%",
                "width": "50%",
              }
            }
          >
            <div
              style={
                Object {
                  "height": "100%",
                  "left": 0,
                  "margin": 0,
                  "padding": 0,
                  "position": "absolute",
                  "top": 0,
                  "width": "100%",
                }
              }
            >
              <div
                style={
                  Object {
                    "backgroundColor": "transparent",
                    "height": 0,
                    "left": 0,
                    "position": "absolute",
                    "top": 0,
                    "width": 0,
                  }
                }
              >
                <div
                  $dimensionKey={0}
                  $geoService={
                    Geo {
                      "hasSize_": false,
                      "hasView_": true,
                      "transform_": Transform {
                        "_maxZoom": 52,
                        "_minZoom": 0,
                        "_zoom": 12,
                        "angle": -0,
                        "center": LatLng {
                          "lat": 42.910972,
                          "lng": -87.881397,
                        },
                        "height": 0,
                        "latRange": Array [
                          -85.05113,
                          85.05113,
                        ],
                        "scale": 4096,
                        "tileSize": 256,
                        "tileZoom": 12,
                        "width": 0,
                        "zoomFraction": 0,
                      },
                    }
                  }
                  $getDimensions={[Function]}
                  $hover={false}
                  $onMouseAllow={[Function]}
                  $prerender={true}
                  lat={42.910972}
                  lng={-87.881397}
                  style={
                    Object {
                      "alignItems": "center",
                      "borderRadius": "50%",
                      "color": "#ff5e58",
                      "display": "flex",
                      "fontSize": "2.2rem",
                      "height": "2.5rem",
                      "justifyContent": "center",
                      "textShadow": "0 1px 0.5rem #fff",
                      "width": "2.5rem",
                    }
                  }
                  text="My Marker"
                >
                  <i
                    className="fas fa-map-marker-alt"
                  />
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="spacer"
      />
      <div
        className="inputContainer"
      >
        <div
          className="root input"
        >
          <input
            placeholder="enter your address..."
            type="text"
            value=""
          />
          <button
            className="root root"
            onClick={null}
            type="button"
          >
            <i
              className="fas fa-map-marker-alt"
            />
          </button>
        </div>
      </div>
    </div>
    <div
      className="color-background--quaternary color--white padding--all"
    >
      <div
        className="max-width--xl margin--horizontal--auto"
      >
        <div
          className="root"
        >
          <div
            className="content"
          >
            Whether you're facing retirement or looking to better understand investments, we can help you with money questions.
          </div>
          <div
            className="action"
          >
            <button
              className="root button"
              onClick={null}
              type="button"
            >
              Get Started Today
            </button>
          </div>
        </div>
        <div
          className="root"
        >
          <div
            className="section"
          >
            <h3>
              Contact
            </h3>
            <div
              className="content"
            >
              <div>
                <a
                  href="mailto:+414-764-7330"
                >
                  414-764-7330
                </a>
              </div>
              <div>
                <a
                  href="https://www.google.com/maps/place/1125+Milwaukee+Ave,+South+Milwaukee,+WI+53172"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  1125 Milwaukee Ave.
                  <br />
                  South Milwaukee
                  , 
                  WI
                   · 
                  53172
                </a>
              </div>
              <div>
                <a
                  href="mailto:social@dbtaxinc.com"
                >
                  social@dbtaxinc.com
                </a>
              </div>
            </div>
          </div>
          <div
            className="section"
          >
            <h3>
              Hours
            </h3>
            <div
              className="content"
            >
              <div
                className="display--flex"
              >
                <div
                  className="day"
                >
                  Sun.
                </div>
                <div>
                  Closed
                </div>
              </div>
              <div
                className="display--flex"
              >
                <div
                  className="day"
                >
                  Mon.
                </div>
                <div>
                  9am-7pm
                </div>
              </div>
              <div
                className="display--flex"
              >
                <div
                  className="day"
                >
                  Tues.
                </div>
                <div>
                  9am-7pm
                </div>
              </div>
              <div
                className="display--flex"
              >
                <div
                  className="day"
                >
                  Wed.
                </div>
                <div>
                  9am-7pm
                </div>
              </div>
              <div
                className="display--flex"
              >
                <div
                  className="day"
                >
                  Thurs.
                </div>
                <div>
                  9am-7pm
                </div>
              </div>
              <div
                className="display--flex"
              >
                <div
                  className="day"
                >
                  Fri.
                </div>
                <div>
                  9am-7pm
                </div>
              </div>
              <div
                className="display--flex"
              >
                <div
                  className="day"
                >
                  Sat.
                </div>
                <div>
                  10am-5pm
                </div>
              </div>
            </div>
          </div>
          <div
            className="section"
          >
            <h3>
              Links
            </h3>
            <div
              className="content"
            >
              <div>
                <a
                  href="#"
                  onClick={[Function]}
                >
                  About
                </a>
              </div>
              <div>
                <a
                  href="#"
                  onClick={[Function]}
                >
                  Services
                </a>
              </div>
              <div>
                <a
                  href="#"
                  onClick={[Function]}
                >
                  Client Resources
                </a>
              </div>
              <div>
                <a
                  href="#"
                  onClick={[Function]}
                >
                  Legal
                </a>
              </div>
              <div>
                <a
                  href="#"
                  onClick={[Function]}
                >
                  Account Access
                </a>
              </div>
            </div>
          </div>
          <div
            className="learnMore"
          >
            <h3>
              Want to Know More?
            </h3>
            <div>
              We'll reach out via email to set up a call.
            </div>
            <div>
              <div
                className="root email"
              >
                <input
                  placeholder="enter your email address..."
                  type="text"
                  value=""
                />
                <button
                  className="root root"
                  onClick={null}
                  type="button"
                >
                  <i
                    className="fas fa-paper-plane"
                  />
                </button>
              </div>
            </div>
          </div>
        </div>
        <div
          className="root"
        >
          <div
            className="logoContainer"
          >
            <a
              href="#"
              onClick={[Function]}
            >
              <img
                alt="DB Tax & Financial Services"
                className="logo"
                src="db-tax-logo-dark.png"
              />
            </a>
          </div>
          <div
            className="copyright"
          >
            ©
            2019
             
            DB Tax & Financial Services
            . All Rights Reserved.
          </div>
          <div
            className="social"
          >
            <button
              className="root root"
              onClick={null}
              type="button"
            >
              <i
                className="fab fa-facebook-f"
              />
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="root"
  >
    <div
      className="root"
      onClick={[Function]}
    />
    <div
      className="menu"
    >
      <a
        aria-current="page"
        className="active"
        href="#"
        onClick={[Function]}
        style={Object {}}
      >
        Home
      </a>
      <a
        aria-current="page"
        className="active"
        href="#"
        onClick={[Function]}
        style={Object {}}
      >
        About
      </a>
      <a
        aria-current="page"
        className="active"
        href="#"
        onClick={[Function]}
        style={Object {}}
      >
        Services
      </a>
      <a
        aria-current="page"
        className="active"
        href="#"
        onClick={[Function]}
        style={Object {}}
      >
        Client Resources
      </a>
      <a
        aria-current="page"
        className="active"
        href="#"
        onClick={[Function]}
        style={Object {}}
      >
        Legal
      </a>
      <a
        aria-current="page"
        className="active"
        href="#"
        onClick={[Function]}
        style={Object {}}
      >
        Account Access
      </a>
      <img
        alt="DB Tax & Financial Services"
        className="logo"
        src="db-tax-logo-icon.png"
      />
    </div>
  </div>
</div>
`;
